docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,106 @@
+---
+title: What to know before onboarding
+sidebar_position: 8
+---
+# What maintainers should know before adopting Packit for downstream release automation 
+
+## Responsibilities
+
+Packit is designed to **assist** maintainers, not to **replace** them.
+
+Maintainers retain key responsibilities:
+- reviewing and merging Packit-created pull requests
+- maintaining package quality
+- ensuring updates don't break dependant packages
+
+:::tip `version_update_mask`
+
+The default Packit onboarding configuration enables automated releases only for `rawhide`. 
+
+When extending the automation to branched Fedoras and EPELs, consider using [`version_update_mask`](https://packit.dev/docs/configuration#version_update_mask) to prevent introducing breaking changes into stable distributions.
+
+:::
+
+:::warning Missing notifications
+
+Pull requests, Koji builds and Bodhi updates are created by Packit FAS account, making it the default notification recipient.
+
+To stay informed about Packit-triggered activities in the Fedora ecosystem, [configure notification settings accordingly](https://packit.dev/docs/fedora-releases-guide/fedora-notifications).
+
+There is an [open issue](https://github.com/packit/packit-service/issues/2404) for Bodhi update notifications.
+
+:::
+
+## Monitoring and observability
```

Comment:
thinking about it more, I think there is quite an overlap with https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding#ui here, what do you think? I would probably try to avoid the duplication

Reply:
You are right, there is some overlap however I think that they make sense in both guides. Or we should merge the missing peaces in the old section and link it in the new guide.

For now I link the old UI and notficiations about errors sections in the new guide.